### PR TITLE
Domains: Update precheck copy when we can't get admin email

### PR DESCRIPTION
**Pull request by aidvu**
If we cannot get admin email, show a different copy to the user.

Test:
- Go to `Domains -> Add`
- Use your own domain
- Pick a domain for transfer, hardcode the email in Calypso to `null`.
- Make sure you see the alternative message

### CODE REVIEW

File: client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx [hunk 1]
```diff
@@ -198,7 +196,31 @@ class TransferDomainPrecheck extends React.PureComponent {
 				},
 			}
 		);
-		const buttonText = translate( 'I can access this email address' );
+		let buttonText = translate( 'I can access the email address' );
```
Comment (aidvu) [inline client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx]: Changed `this` to `the` as there's no email address shown for no email.

File: client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx [hunk 2]
```diff
@@ -179,15 +179,13 @@ class TransferDomainPrecheck extends React.PureComponent {
 		const isStepFinished = currentStep > step;
 
 		const heading = translate( 'Verify we can get in touch.' );
-		const message = translate(
-			"We'll send an email to {{strong}}%(email)s{{/strong}} to start the transfer process. Make sure " +
-				"you have access to that address. Don't recognize it? Then you have privacy protection enabled. " +
-				"You'll need to log in to your current domain provider and {{a}}turn it off{{/a}} before we start. " +
-				"Don't worry, you can re-enable it once the transfer is done.",
+		let message = translate(
```
Comment (aidvu) [inline client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx]: I'd even be honest here and say we were unable to get the e-mail, so they should check bla bla. What do you think?

File: client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx [hunk 3]
```diff
@@ -179,15 +179,13 @@ class TransferDomainPrecheck extends React.PureComponent {
 		const isStepFinished = currentStep > step;
 
 		const heading = translate( 'Verify we can get in touch.' );
-		const message = translate(
-			"We'll send an email to {{strong}}%(email)s{{/strong}} to start the transfer process. Make sure " +
-				"you have access to that address. Don't recognize it? Then you have privacy protection enabled. " +
-				"You'll need to log in to your current domain provider and {{a}}turn it off{{/a}} before we start. " +
+		let message = translate(
```
Comment (aidvu) [inline client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx]: I'd even be honest here and say we were unable to get the e-mail, so they should check bla bla. What do you think?

File: client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx (hunk 1 above)
Comment (a8ci18n) [inline client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx]: :ok: This change is safe, we'll keep the existing 18 translations.

File: client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx (hunk 3 above)
Comment (danhauk) [reply client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx]: +1 I was just thinking the same thing.

**Comment by danhauk** [comment]
@aidvu I pushed up some more copy edits. I put the main point we need as the first sentence in both cases. The reason for this is to hopefully account for users not reading much and getting the message across as quickly as possible, with more explanation for those who do need it.

Good idea explicitly stating we couldn't get the email address.

If my commit looks good to you feel free to merge. 👍

**Pull request merged**